### DISCUSSION TITLE
Is Z-Stack_3.x.0 20220219 unstable?

### DISCUSSION THREAD

**Discussion by okastl** [Koenkk/zigbee2mqtt#12083] [General]
I recently upgraded my Slaesh CC2652RB coordinator stick from Z-Stack_3.x.0 20211217 to Z-Stack_3.x.0 20220219. After 1-2 days operation, the stick "hangs". I cannot send or receive messages from the zigbee network. This happened twice now. I have reverted to 20211217 for now to observe.
I am using z2m [1.25.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.25.0) commit: [6f1460e4](https://github.com/Koenkk/zigbee2mqtt/commit/6f1460e4)

This could be just a coincidence, hardware failure, but it could be a problem with 20220219.

I just want to ask, has anybody seen similar instability with Z-Stack_3.x.0 20220219 coordinator?

**Comment by c0mplex1** [comment]
Hi,

On April 1, I downloaded and installed the Z-stack_3.x.0 20220219 firmware on my CC2652RB stick.
So far I have not encountered any problems.
